sunpy/net/dataretriever/sources/noaa.py

```diff
@@ -204,11 +206,11 @@ def fetch(self, qres, path=None, error_callback=None, **kwargs):
         filenames = []
         local_filenames = []
 
-        for i, [url, qre] in enumerate(zip(urls, qres)):
+        for i, url in enumerate(urls):
             name = url.split('/')[-1]
 
             # temporary fix !!! coz All QRBs have same start_time values
```

Comment:
Maybe it is time to remove this?

Reply:
do you mean the loop here? 

This client could actually do with being refactored

Reply:
I meant just the comment.

Reply:
oh sorry yes - removed now